sources/packages/forms/src/form-definitions/reportscholasticstandingchange.json

```diff
@@ -6,16 +6,264 @@
     "owner": "andrew.schwenker@gov.bc.ca",
     "components": [
         {
-            "title": "Application Details Panel",
-            "theme": "default",
+            "label": "Student Name",
+            "labelWidth": "",
+            "labelMargin": "",
+            "tag": "p",
+            "className": "",
+            "attrs": [
+                {
+                    "attr": "",
+                    "value": ""
+                }
+            ],
+            "content": "<span class=\"category-header-large primary-color mr-2\">{{ data.applicationStudentName }} </span> \n<span class=\"fa fa-circle form-status-badge status-resolved-restriction py-1\"><span class=\"mx-1\">\n{{ data.applicationStatus }} </span></span>",
```

Comment:
For the status chip please reuse the logic previously done for the Designation Agreement.
```html
<span class="fa fa-circle formio-status-chip {{ data.designationStatusClass }}">{{ data.designationStatus }}</span>
```

Reply:
mm actually I tried this, but it was not working for me. I will try again

Reply:
actually, we have 2 classes for formio status chip, 
![image](https://user-images.githubusercontent.com/77353155/163866175-cd92ff9a-bfc5-4b20-910a-c5459d43235d.png), i used here `form-status-badge`

Reply:
there is an extra class that is not required. i removed it

Reply:
so, which class are we keeping

Reply:
i will use this `formio-status-chip`. with this i can avoid some extra bootstrap classes for padding

Reply:
![image](https://user-images.githubusercontent.com/77353155/163867515-4584a30c-8b1c-4c6b-b097-4d53badcd410.png)
1 is `form-status-badge` and 2 is `formio-status-chip`, 1st classes is matching with the Figma more 
![image](https://user-images.githubusercontent.com/77353155/163867653-c5fd4ad3-a3f1-4726-98a8-5eb3ea0447ea.png)

Reply:
added  `formio-status-chip`